### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- a  couple of minor style fixes
- fix http bug (wrong state if action listen is not confirmed by api user) #42 
- small dropout in keras model to make it overfit a bit less

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog

### CODE REVIEW

File: rasa_core/policies/keras_policy.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def _build_model(self, num_features, num_actions, max_history_len):
         # Build Model
         model = Sequential()
         model.add(Masking(-1, batch_input_shape=batch_shape))
-        model.add(LSTM(n_hidden, batch_input_shape=batch_shape))
+        model.add(LSTM(n_hidden, batch_input_shape=batch_shape, dropout=0.2))
```
Comment (amn41) [inline rasa_core/policies/keras_policy.py]: please double check whether this means "keep 20%" or "drop 20%" - varies between DL frameworks

Comment (tmbo) [reply rasa_core/policies/keras_policy.py]: that is the fraction to drop
